CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### Added
+
+- Lithology csv import.
+
```

Comment:
REQ: Diesen Block nach [Unreleased] schieben, da wir in der Zwischenzeit BDMS released haben.